packages/rum/test/benchmarks/server.js

```diff
@@ -25,38 +25,44 @@
 
 const express = require('express')
 const path = require('path')
-const { elasticApmUrl, port } = require('./config')
-const { createReadStream } = require('fs')
+const { port, imageCount } = require('./config')
+const { getMinifiedApmBundle } = require('./analyzer')
 
 const pages = path.join(__dirname, 'pages')
-const dist = path.join(__dirname, '../../dist')
 
-function generateImageUrls(number) {
+function generateImageUrls() {
```

Comment:
nit: I still think this function can be a pure function and only rely on its parameters instead of using something from the other scope. 

As a general good practice I try to make pure functions when possible and defer using global scope to the caller function.

Reply:
Yeah sounds good, I made it a config since we need to add it `parameters.images`. just refactored the code a bit as you mentioned.